started/index.md

```diff
@@ -98,6 +102,16 @@ Note that these steps are just required for development - your Fyne applications
 </div>
 </div>
 
+<div id="install__bsd" class="hidden">
+<div style="text-align: left" markdown="1">
+
+* You will need to install Go, gcc and the graphics library header files using the package manager.
+* **FreeBSD:**
+* `sudo pkg install go gcc xorg glfw pkgconf`
```

Comment:
glfw should not be needed

Reply:
I uninstalled `glfw` and could not build without it. I'd say that it seems to be needed.

Reply:
Installing again and recompiling works.

Reply:
Can you share the error please? It should not be required as we embed the code.
Are you sure it's not one of the dependencies it brings in instead?

Reply:
It only mentions installing `glfw` and nothing more. Hmm, interesting. I opened the binary that I had compiled before removing `glfw` and got this:
```
ld-elf.so.1: Shared object "libglfw.so.3" nit found, required by "fyne_demo"
```

Getting this if I remove th binary and recompile:
```
/usr/local/go/pkg/tool/freebsd_amd64/link: running clang failed: exit status 1
ld: error: unable to find library -lglfw
```

If I compile with `go build -a` it will fail with `pkg-config` complaining that it can't find `glfw3`.

Reply:
No way! OK, can you provide the error for the last case, it should point us to which file requires the library that I thought we were embedding.

Reply:
Out of interest, I opened the cross-compiled FreeBSD binary build using `fyne-cross` and it also won't open without glfw installed. Should I open an issue in `fyne-io/fyne` about shared FreeBSD compiles?

Sorry, but I don't think the last command gives a whole lot of more information:
```
[jacob@jacob-ghostbsd-pc ~/go/src/fyne.io/fyne/v2/cmd/fyne_demo]$ go build -a
# pkg-config --cflags  -- glfw3
Package glfw3 was not found in the pkg-config search path.
Perhaps you should add the directory containing `glfw3.pc'
to the PKG_CONFIG_PATH environment variable
Package 'glfw3', required by 'virtual:world', not found
pkg-config: exit status 1
```